protocol/src/main/java/org/geysermc/mcprotocollib/network/crypt/PacketEncryption.java

```diff
@@ -28,10 +13,9 @@ public interface PacketEncryption {
      * @param inputLength Length of the data to be decrypted.
      * @param output Array to output decrypted data to.
      * @param outputOffset Offset of the output array to start at.
-     * @return The number of bytes stored in the output array.
      * @throws Exception If an error occurs.
      */
-    int decrypt(byte[] input, int inputOffset, int inputLength, byte[] output, int outputOffset) throws Exception;
+    void decrypt(byte[] input, int inputOffset, int inputLength, byte[] output, int outputOffset) throws Exception;
 
     /**
      * Encrypts the given data.
```

Comment:
mention that the input and output arrays may be the same instance, in which case the read and write regions may overlap

Reply:
No, with AES/CFB8/NoPadding input size == output size